guides/source/form_helpers.md

```diff
@@ -79,15 +79,7 @@ IMPORTANT: Use "GET" as the method for search forms. This allows users to bookma
 
 ### Helpers for Generating Form Elements
 
-Rails provides a series of helpers for generating form elements such as
-checkboxes, text fields, and radio buttons. These basic helpers, with names
-ending in `_tag` (such as `text_field_tag` and `check_box_tag`), generate just a
-single `<input>` element. The first parameter to these is always the name of the
-input. When the form is submitted, the name will be passed along with the form
-data, and will make its way to the `params` in the controller with the
-value entered by the user for that field. For example, if the form contains
-`<%= text_field_tag(:query) %>`, then you would be able to get the value of this
-field in the controller with `params[:query]`.
+The form builder object yielded by `form_with` provides numerous helper methods for generating form elements such as text fields, checkboxes, and radio buttons. The first parameter to these methods is always the name of the input. When the form is submitted, the name will be passed along with the form data, and will make its way to the `params` in the controller with the value entered by the user for that field. For example, if the form contains `<%= form.text_field :query %>`, then you would be able to get the value of this field in the controller with `params[:query]`.
```

Comment:
Let's retain the multi-line if possible(unless this was visually affecting something).
It's easier to read/change in future than a single line + better for diffs.

Reply:
I think git's intra-line diff is actually pretty good for long lines ([example](https://github.com/rails/rails/pull/39344/files#diff-6c86ada7eb016a7b145c128714a8d379R111), [example](https://github.com/rails/rails/pull/39344/files#diff-6c86ada7eb016a7b145c128714a8d379R337)), but I do agree we should try to minimize diffs.  I changed this to only unwrap the lines that I completely rewrote.